win32/readjustService.ps1

```diff
@@ -35,7 +35,7 @@ function doAdjust_ACmode {
     $Script:repeatWaitTimeSeconds = 1    #only use values below 5s if you are using $monitorField
     adjust "fast_limit" 46000
     adjust "slow_limit" 25000
-    #adjust "slow_time" 30
+    #adjust "coall" 4294967266    #Curve optimizer at -30
```

Comment:
I didn't check this parameter, but does the CLI version also require unsigned int?
Would it be possible to use a signed int and directly use -30?

Reply:
@asymcon could you please change it to signed int to be able to use -30 directly?